### PR TITLE
Some ava tweaks

### PR DESCRIPTION
**Pull request by andrepolischuk**

### CODE REVIEW

File: app/templates/_test-ava.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import test from 'ava';
 import <%= camelModuleName %> from './index';
 
 test('should <%= camelModuleName %>', (t) =>
-  t.same(<%= camelModuleName %>('unicorns'), 'unicorns'));
+  t.is(<%= camelModuleName %>('unicorns'), 'unicorns'));
 
 test('should <%= camelModuleName %> invalid input', (t) =>
-  t.same(<%= camelModuleName %>(), undefined));
+  t.is(<%= camelModuleName %>(), undefined));
```
Comment (iamstarkov) [inline app/templates/_test-ava.js]: why?

File: app/test-frameworks.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ var testFrameworksHash = {
   ava: {
     test: 'ava --require babel-register',
     tdd: 'npm test -- --watch',
-    deps: ['ava@^0.12.0'],
+    deps: ['ava@^0.13.0'],
```
Comment (iamstarkov) [inline app/test-frameworks.js]: 👍

File: app/templates/_test-ava.js (hunk 1 above)
Comment (andrepolischuk) [inline app/templates/_test-ava.js]: `t.same` is a deep equalization. Why use it for primitive values?

You don't use `.deepEqual` method in `mocha` and `tape` templates.

Comment (iamstarkov) [inline app/templates/_test-ava.js]: You are right

**Comment by iamstarkov** [comment]
Thanks

**Pull request merged**